app/src/main/java/com/nervousfish/nervousfish/ConstantKeywords.java

```diff
@@ -12,10 +13,10 @@
     public static final String PROXY_REQUIRED = "Proxy required.";
 
     /**
-     * Prevent instantiation
+     * Prevents instantiation from outside the class
      */
     private ConstantKeywords() {
-
+        // Unused, just prevents instantiation from outside the class
```

Comment:
I know, but I'm pretty sure we disabled that rule (given that this constructor used to be empty) and otherwise maybe we should 😉